### PR TITLE
[action] is_ci also looks at GITHUB_ACTIONS

### PR DESCRIPTION
**Pull request by koenpunt**
GitHub CI doesn't always have a `GITHUB_ACTION` environment variable, but it does have `GITHUB_ACTIONS` set to `true`.

### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### CODE REVIEW

**Review by joshdholtz — COMMENTED** [review]
This makes sense 🙃 Do you happen to know where the documentation for this is? Or have an example output? I don't see anything on https://developer.github.com/actions/creating-github-actions/accessing-the-runtime-environment/#environment-variables about `GITHUB_ACTIONS`. Just curious before smashing the merge button 😊

**Comment by koenpunt** [comment]
I simply did run `env` in a workflow, and noticed there being a `GITHUB_ACTIONS`, but not `GITHUB_ACTION`. But now that I look again I actually see both.. 
So I'm not too sure of this PR anymore..

**Comment by koenpunt** [comment]
Btw, the new location for the environment variable documentation is here: https://help.github.com/en/articles/virtual-environments-for-github-actions#environment-variables

**Comment by joshdholtz** [comment]
@koenpunt How about if we put both `GITHUB_ACTION` and `GITHUB_ACTIONS` in here? 😇 Doesn't really hurt and that way we know if will 100% work

**Comment by koenpunt** [comment]
Feel free to squash and merge when the build succeeds.

**Review by joshdholtz — APPROVED** [review]
Thanks for adjusting that and thanks for the contribution ❤️

**Pull request merged**
